data/gen_data.py

```
@@ -94,40 +94,43 @@
 
+
 def translit_to_cyrillic(name):
     name = normalize_apostrophe(name)
 
     # Спробуємо просту заміну (тільки для латиниці)
-    res = ''
+    res = ""
     i = 0
     while i < len(name):
         # Пробуємо знайти довгі комбінації
         for l in [4, 3, 2, 1]:
```

Ruff (0.14.14)

[error] Ambiguous variable name: `l`

(E741)